plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingAltinityServer.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.clickhouse;
+
+import org.testcontainers.containers.ClickHouseContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+import static org.testcontainers.containers.ClickHouseContainer.HTTP_PORT;
+import static org.testcontainers.utility.MountableFile.forClasspathResource;
+
+/**
+ * @see <a href="https://docs.altinity.com/altinitystablebuilds/">Altinity Stable Builds</a>
+ */
+public class TestingAltinityServer
```

Comment:
What's the difference between this class and TestingClickHouseServer?

Is it only in docker image name/versions being used?
Why not reuse `TestingClickHouseServer`?